content/dstu2/contract.md

```diff
@@ -42,11 +42,11 @@ Notes:
 ### Response
 
 Search for all Contracts subject to a specific patient
-<%= headers 200, GET: '[...]/Contract?subject:Patient=14067775' %>
+<%= headers 200, GET: '[...]/Contract?subject:Patient=4598008' %>
```

Comment:
Also looks like the existing Contract JSON example is using FHIRPLAY, is that necessary @cerbs ?